Lib/cu2qu/rf.py

```diff
@@ -48,29 +53,35 @@ def fonts_to_quadratic(*fonts, **kwargs):
     font at a time may yield slightly more optimized results.
     """
 
-    report = kwargs.get('report', {})
-    dump_report = kwargs.get('dump_report', False)
-    max_n = kwargs.get('max_n', 10)
+    if stats is None:
+        stats = {}
 
-    max_err_em = kwargs.get('max_err_em', 0.0025)
-    max_err = kwargs.get('max_err', None)
+    if max_err_em and max_err:
+        raise TypeError('Only one of max_err and max_err_em can be specified.')
+    if not (max_err_em or max_err):
+        max_err_em = DEFAULT_MAX_ERR
     if max_err:
         max_errors = [max_err] * len(fonts)
     else:
         max_errors = [f.info.unitsPerEm * max_err_em for f in fonts]
 
-    for glyph in FontCollection(fonts):
-        glyph_to_quadratic(glyph, max_errors, max_n, report)
+    if len(fonts) == 1:
```

Comment:
Humm.  Two things:

* I wasn't suggesting that it doees wrap now, but that it can and should wrap,

* I suppose not using a FontCollection for the case of one font might be measurably faster, but it makes me a bit uneasy about having tw oseparate codepaths for the two usecases.  What do others think?